### PR TITLE
[simdjson] Update to 3.1.5

### PR DESCRIPTION
**Pull request by myd7349**
- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md)
- [x] SHA512s are updated for each updated download
- [ ] ~~The "supports" clause reflects platforms that may be fixed by this new version~~
- [ ] ~~Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.~~
- [ ] ~~Any patches that are no longer applied are deleted from the port's directory.~~
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

### CODE REVIEW

**Comment by jimwang118** [comment]
I checked that the latest version of `simdjson` is 3.1.5, please update to the latest version.

**Comment by myd7349** [comment]
Thanks. @jimwang118 It seems simdjson has two new releases in the past 24 hours.

**Comment by jimwang118** [comment]
All feature passed with following triplets:

x86-windows
x64-windows
x64-windows-static

File: ports/simdjson/portfile.cmake [hunk 1]
```diff
@@ -1,9 +1,9 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO simdjson/simdjson
-    REF 8a247f13216aa5a0b149ebb5e352d26eb13f309a # v3.1.2
+    REF v${VERSION}
```
Comment (Cheney-W) [inline ports/simdjson/portfile.cmake:4]:
```suggestion
    REF "v${VERSION}"
```

Comment (myd7349) [reply ports/simdjson/portfile.cmake]: Thanks. @Cheney-W Updated.

**Pull request merged**